outputs/openapi/schema.go

```diff
@@ -32,10 +35,10 @@ func mapParamToSchema(param astra.Param) Schema {
 	}
 }
 
-func mapFieldToSchema(field astra.Field) Schema {
+func mapFieldToSchema(bindingType astTraversal.BindingTagType, field astra.Field) Schema {
```

Comment:
Might be an idea to "servicify" these soon if you know what I mean (use a struct to maintain params and make the map funcs to become methods)
But not in scope

Reply:
Yeah I think that would be useful but out of scope for the moment.